compute_tools/src/spec_apply.rs

```diff
@@ -703,9 +710,13 @@ async fn get_operations<'a>(
                                 )
                             } else {
                                 format!(
-                                    "CREATE ROLE {} {}",
+                                    // GenericOptions are explicitly ignored for JWKS roles.
+                                    // JWKS roles are ment to have minimal privileges.
+                                    // The ROLE clause causes one or more specified existing roles
+                                    // to be automatically added as members
+                                    "CREATE ROLE {} ROLE {}",
                                     role.name.pg_quote(),
-                                    role.to_pg_options(),
+                                    db_owners.join(","),
```

Comment:
I don't think we can do it due to the same reason we discussed -- there are zero guarantees that in general case all `db_owners` are present in the state this SQL command expects them to be

This is also strictly against the statement we agreed on last week:

> we need 3 distinctive RLS roles: authenticator, authenticated, and anonymous -- it's not expected that they will intersect in any way with normal Postgres role managed by user, we also need to mark these 3 roles protected, so that user wasn't able to mess this up

Here I see that we put some explicit dependencies between RLS roles and normal user-managed roles. So I'm confused again. This doesn't look like something we are ready to commit.